torchvision/datasets/vggface2.py

```diff
@@ -0,0 +1,160 @@
+            author    = "Cao, Q. and Shen, L. and Xie, W. and Parkhi, O. M. and Zisserman, A.",
+            title     = "VGGFace2: A dataset for recognising faces across pose and age",
+            booktitle = "International Conference on Automatic Face and Gesture Recognition",
+            year      = "2018"}
+
+        Args:
+            root (string): Root directory of the VGGFace2 Dataset.
+            Expects the following folder structure if download=False:
+                .
+                └── vggface2
+                    ├── bb_landmark.tar.gz (or 'bb_landmark' if uncompressed)
+                    ├── vggface2_train.tar.gz (or 'train' if uncompressed)
+                    ├── vggface2_test.tar.gz (or 'test' if uncompressed)
+                    ├── train_list.txt
+                    └── test_list.txt
+            split (string): The dataset split to use. One of {``train``, ``test``}.
+                Defaults to ``train``.
+            target_type (string): The type of target to use. One of
+                {``class_id``, ``image_id``, ``face_id``, ``bbox``, ``landmarks``.``""``}
+                Can also be a list to output a tuple with all specified target types.
+                The targets represent:
+                    ``class_id``  (string)
+                    ``image_id``  (string)
+                    ``face_id``   (string)
+                    ``bbox``      (torch.tensor shape=(4,) dtype=int): bounding box (x, y, width, height)
+                    ``landmarks`` (torch.tensor shape=(10,) dtype=float): values that
+                        represent five points (P1X, P1Y, P2X, P2Y, P3X, P3Y, P4X, P4Y, P5X, P5Y)
+                Defaults to ``bbox``. If empty, ``None`` will be returned as target.
+            transform (callable, optional): A function/transform that  takes in a PIL image
+                and returns a transformed version. E.g, ``transforms.RandomCrop``
+            target_transform (callable, optional): A function/transform that takes in the
+                target and transforms it.
+            download (bool, optional): If true, downloads the dataset from the internet and
+                puts it in root directory. If dataset is already downloaded, it is not
+                downloaded again.
+        """
+
+    base_folder = "vggface2"
+    file_list = [
+        # Filename                MD5 Hash                            Uncompressed filename
+        ("vggface2_train.tar.gz", "88813c6b15de58afc8fa75ea83361d7f", "train"),
+        ("vggface2_test.tar.gz", "bb7a323824d1004e14e00c23974facd3", "test"),
+        ("bb_landmark.tar.gz", "26f7ba288a782862d137348a1cb97540", "bb_landmark")
+    ]
+
+    def __init__(
+        self,
+        root: str,
+        split: str = "train",
+        target_type: Union[List[str], str] = "bbox",
+        transform: Optional[Callable] = None,
+        target_transform: Optional[Callable] = None,
+        download: bool = False,
+    ) -> None:
+        import pandas
```

Comment:
Do we really need to use pandas? I'm not sure if its worth importing a new library just for `concat`. We should be fine with just a dict here